src/main/java/io/xstefank/wildlfy/bot/TriagePullRequestProcessor.java

```diff
@@ -28,12 +62,17 @@ void onPullRequestOpened(@PullRequest.Opened GHEventPayload.PullRequest pullRequ
         Set<String> mentions = new TreeSet<>();
 
         for (WildFlyRule rule : wildflyBotConfigFile.wildfly.rules) {
-            if (Matcher.matches(pullRequest, rule)) {
-                for (String nick : rule.notify) {
-                    if (!nick.equals(pullRequest.getUser().getLogin())) {
-                        mentions.add(nick);
+            if (rule.id != null) {
+                if (Matcher.matches(pullRequest, rule)) {
+                    LOG.debugf("Matched rule with id: %s.", rule.id);
+                    for (String nick : rule.notify) {
+                        if (!nick.equals(pullRequest.getUser().getLogin())) {
+                            mentions.add(nick);
+                        }
                     }
                 }
+            } else {
+                LOG.warn("Rule is missing an id.");
```

Comment:
So we need to decide if it's an error or a warning. Do we need to log message on every PR processing? I think that on startup is enough. Well, we should also execute the (current) postconstruct method when a new repo registers. Can you add that please?

Reply:
Should I open a new issue for that?

Reply:
No, just remove it here to remove duplications